pkgs/by-name/ba/bash-pinyin-completion-rs/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "bash-pinyin-completion-rs";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "AOSC-Dev";
+    repo = "bash-pinyin-completion-rs";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-h4l4plGMn5WMhU60+m60Uf45UfPNDb0X+E2LK3U3jxw=";
+  };
+
+  strictDeps = true;
+
+  cargoHash = "sha256-SAegFsmn91xrWg0o7lHgk+vRqTQhabev9dP+Lbk/h5s=";
+
+  postInstall = ''
+    substituteInPlace scripts/bash_pinyin_completion \
+      --replace-fail 'bash-pinyin-completion-rs' '$out/bin/bash-pinyin-completion-rs' \
```

```suggestion
      --replace-fail 'bash-pinyin-completion-rs' "$out/bin/bash-pinyin-completion-rs" \
```

I corrected this in my suggestion shortly afterwards